presto-main/src/main/java/io/prestosql/operator/ExchangeClientFactory.java

```diff
@@ -96,6 +110,25 @@ public ExchangeClientFactory(
         checkArgument(concurrentRequestMultiplier > 0, "concurrentRequestMultiplier must be at least 1: %s", concurrentRequestMultiplier);
     }
 
+    public ExchangeClientFactory(
+            DataSize maxBufferedBytes,
+            DataSize maxResponseSize,
+            int concurrentRequestMultiplier,
+            Duration maxErrorDuration,
+            boolean acknowledgePages,
+            int pageBufferClientMaxCallbackThreads,
+            HttpClient httpClient,
+            ScheduledExecutorService scheduler,
+            FailureDetector failureDetector)
+    {
+        this(maxBufferedBytes, maxResponseSize,
+                concurrentRequestMultiplier,
+                maxErrorDuration,
+                acknowledgePages,
+                pageBufferClientMaxCallbackThreads,
+                httpClient, scheduler, failureDetector, true, 10);
```

Comment:
lets not hardcode this value to 10. can take default value

Reply:
fixed